plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineFlexFacet.java

```diff
@@ -35,7 +35,7 @@
    * @param project should not be null
    * @return true if project has the App Engine Flex facet and false otherwise
    */
-  public static boolean hasAppEngineFacet(IFacetedProject project) {
+  public static boolean hasFacet(IFacetedProject project) {
```

Comment:
There is a counterpart in `AppEngineStandardFacet`, so please rename the method there too.